Timer/CMakeLists.txt

```diff
@@ -39,6 +39,8 @@ find_package(IARMBus)
 
 target_include_directories(${MODULE_NAME} PRIVATE ${IARMBUS_INCLUDE_DIRS} ../helpers)
 
+set_source_files_properties(Timer.cpp PROPERTIES COMPILE_FLAGS "-fexceptions")
```

Comment:
Any reason why this is needed?

Reply:
Here is the error I got without that line, I followed in similar lines as AVInput and add it to Timer.

> 
| In file included from /Users/svemur170/Dev/rdkservices/rdkservices/Timer/Timer.cpp:28:
| /Users/svemur170/Dev/rdkservices/rdkservices/Timer/Timer.cpp: In member function 'uint32_t WPEFramework::Plugin::Timer::cancelWrapper(const JsonObject&, JsonObject&)':
| /Users/svemur170/Dev/rdkservices/rdkservices/Timer/../helpers/UtilsJsonRpc.h:48:16: error: exception handling disabled, use '-fexceptions' to enable
|    48 |         catch (...) { param = 0; } \
|       |                ^~~
| /Users/svemur170/Dev/rdkservices/rdkservices/Timer/Timer.cpp:334:13: note: in expansion of macro 'getNumberParameter'
|   334 |             getNumberParameter("timerId", timerId);
|       |             ^~~~~~~~~~~~~~~~~~
| make[2]: *** [Timer/CMakeFiles/WPEFrameworkTimer.dir/build.make:66: Timer/CMakeFiles/WPEFrameworkTimer.dir/Timer.cpp.o] Error 1
| make[1]: *** [CMakeFiles/Makefile2:531: Timer/CMakeFiles/WPEFrameworkTimer.dir/all] Error 2
| make: *** [Makefile:133: all] Error 2
[unit-tests/all]   ❌  Failure - Build rdkservices

>